src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudTemplateIT.java

```diff
@@ -131,4 +139,42 @@ public void testTemplateCloudIdLabelKeyAndValue() {
     assertEquals(
         cloud.getInstanceId(), instance.getLabels().get(ComputeEngineCloud.CLOUD_ID_LABEL_KEY));
   }
+
+  @Test
+  public void testConnectionWithTemplateSshKey() throws IOException, Exception {
+    assertTrue(connectToInstance());
+  }
+
+  private boolean connectToInstance() throws IOException, Exception {
```

Comment:
Since this is only called from the test method above, I think it makes sense to in-line it within the test method itself.